index.rst

```diff
@@ -348,7 +354,7 @@ If working with an individual CSC, which should be a special case, the ``salobj.
     await atdome.start_task()
 
     await salobj.set_summary_state(
-    atdome, salobj.State.ENABLED, configurationOverride="simple_algorithm.yaml"
+    atdome, salobj.State.ENABLED, override="simple_algorithm.yaml"
```

Comment:
This code example is out of date. The ATDome does not provide this as an override file.

Reply:
see comment below